### PR TITLE
Added my kimai clients to marketplace

### PR DESCRIPTION
**Pull request by infeeeee**
I hope I added everything required.

I'm not sure if I can use gifs as screenshots.

On the kimai1 website plugins had icons showing which os they are running on, do you plan to add this? Maybe next to the clients name as a fontawesome icon on the list. Maybe I can try to add this later.

### CODE REVIEW

File: _store/infeeeee-kimai2-cmd.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+github: https://github.com/infeeeee/kimai2-cmd
+tags:
+  - client
+---
+
+Command line client for Kimai2
+
+![interactive start gif](https://raw.githubusercontent.com/infeeeee/kimai2-cmd/master/assets/interactive-start.gif)
+
+## Fetaures
```
Comment (kevinpapst) [inline _store/infeeeee-kimai2-cmd.md]: Features?

**Comment by kevinpapst** [comment]
I just gave it a try, looks great.
Do you want to fix that one typo in the heading before merging?

**Comment by kevinpapst** [comment]
Ups, accidentally pushed a docu to your branch... 😁

**Comment by kevinpapst** [comment]
Don't want to change the history of the branch, so I am going to merge altgether...

File: _documentation/fresh-ubuntu-18.md [hunk 2]
```diff
@@ -0,0 +1,239 @@
...
+And restart the SSH Daemon:
+```bash
+/etc/init.d/ssh restart
+```
+
+## Install PHP, webserver and database
+
+Lets start with all required software:
+```bash
+apt-get update
```
Comment (infeeeee) [inline _documentation/fresh-ubuntu-18.md]: `apt-get` is the old form, nowadays simply `apt` is enough.

Comment (kevinpapst) [reply _documentation/fresh-ubuntu-18.md]: Ha, didn't know that. Thanks for the info. Will try it next time

Comment (infeeeee) [reply _documentation/fresh-ubuntu-18.md]: You also get a nice progress bar with apt, so it's definitely better!

**Pull request merged**